integreat_cms/api/v3/events.py

```diff
@@ -152,12 +157,18 @@ def events(request, region_slug, language_slug):
     """
     region = request.region
     result = []
+    language = region.get_language_or_404(language_slug, only_active=True)
     now = timezone.now().date()
     for event in region.events.prefetch_public_translations().filter(archived=False):
         event_translation = event.get_public_translation(language_slug)
+        if not event_translation and region.use_fallback_language:
+            event_translation = event.get_public_translation(
+                region.default_language.slug
+            )
```

Comment:
If we do not set the language attribute of the fallback translation, then the link will be incorrect in the API response.
What problems exactly did you face with the "fake" language?
E.g. for the test data, the endpoint `http://localhost:8000/api/augsburg/ar/events/` should return the test event with the "fake" url `http://localhost:8000/augsburg/ar/events/test-veranstaltung/` and the available languages `de`, `en` and `fa` where the link/path always points to the available language, no matter whether it's a real translation or a fake one:
```
"id": 2,
"url": "http://localhost:8000/augsburg/ar/events/test-veranstaltung/",
"path": "/augsburg/ar/events/test-veranstaltung/",

...

"available_languages": {
    "de": {
        "id": 2,
        "url": "http://localhost:8000/augsburg/de/events/test-veranstaltung/",
        "path": "/augsburg/de/events/test-veranstaltung/"
    },
    "en": {
        "id": 3,
        "url": "http://localhost:8000/augsburg/en/events/test-event/",
        "path": "/augsburg/en/events/test-event/"
    },
    "fa": {
        "id": 2,
        "url": "http://localhost:8000/augsburg/fa/events/test-veranstaltung/",
        "path": "/augsburg/fa/events/test-veranstaltung/"
    },
},
```
I don't know yet what to do about the ids, but since the app is not using them anyway, I think it's not that important that the ids make sense.